docker/config_template.yaml

```diff
@@ -282,6 +282,12 @@ global:
         prometheus:
             timerType: {{ default .Env.PROMETHEUS_TIMER_TYPE "histogram" }}
             listenAddress: "{{ .Env.PROMETHEUS_ENDPOINT }}"
+            {{- if .Env.PROMETHEUS_PREFIX }}
+            prefix: {{ .Env.PROMETHEUS_PREFIX }}
+            {{- end }}
+            {{- if .Env.PROMETHEUS_TAGS }}
+            tags: {{ .Env.PROMETHEUS_TAGS }}
+            {{- end }}
```

Comment:
How does the prefix and tags wire up when init the prometheus config?

Reply:
oh, I misplaced those 🤦 , they should be in `metrics`, not `prometheus` itself.
I'll fix that.

Now I wonder whether I should make these new options available for both prometheus and statsd. Something like `METRICS_PREFIX` and `METRICS_TAGS`. 🤔
Any thoughts?

Reply:
Changed. 🙏